metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/configuration/MetadataDatabaseConfiguration.java

```diff
@@ -45,6 +46,7 @@
         basePackages = "uk.ac.ebi.ampt2d.metadata.persistence.repositories",
         transactionManagerRef = "metadataTransactionManager")
 @EnableJpaAuditing
+@EnableRetry
```

Comment:
do we need @EnableRetry here as well?

Reply:
We don't, thanks! Removed in https://github.com/EBIvariation/metadata-ws/pull/72/commits/696e5da9cda2aa8d75532c11cd275afe554d01a9